### DISCUSSION TITLE
Feature Request: Support code cross-references for Bazel projects

### DISCUSSION THREAD

**Discussion by davidzchen** [community/community#114305] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

There are many projects, such as TensorFlow and Abseil, that widely used and built with Bazel. However, GitHub does not seem to support Bazel in code cross references.

Would it be possible to implement support for Bazel projects for cross references?

**Comment by look** [comment, marked as answer]
@davidzchen thank you for the feedback. Can you clarify what you have in mind and give some examples? 

I checked TensorFlow and for example, [this BUILD file](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/BUILD) supports code navigation.

**Comment by davidzchen** [comment]
Hi @look, thank you for getting back to me so quickly.

You're right. I checked again, and it looks like xrefs do indeed work for these two projects. Is this feature limited to public repos? I have a private C++ repository built with Bazel, but clicking on most symbols does not bring up its definition.

**Reply by look** [reply]
@davidzchen your repository may not be indexed yet. Try doing a search on it. If it fits within our size and content limits, this should trigger indexing and it will be searchable within about 5 minutes. Once it's indexed, the code navigation should become available. If that doesn't work, if you're comfortable sharing the repo name with me, I can look it up in our administrative tooling.

**Reply by davidzchen** [reply]
Thanks! Is there a way I can message you directly to provide you with the repository names?

**Reply by look** [reply]
You can file a support request and ask them to direct it to me. Or just email me (look@).
